apps/docs/content/guides/auth/auth-mfa.mdx

```diff
@@ -260,13 +144,15 @@ export function UnenrollMFA() {
           <td>Factor ID</td>
           <td>Friendly Name</td>
           <td>Factor Status</td>
+          <td>Phone Number</td>
```

Comment:
For TOTP factors, this will be empty. I considered adding an alternate placeholder value such as N.A. but figured that would be harder to read